config/Dockerfiles/tests.d/combined/01_beaker_openstack

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -xe
 
 # Provision both beaker and openstack with one PinFile
-# distros.exclude: none
-# providers.include: beaker
+# distros.exclude: fedora27 fedora26 centos7
+# providers.include: none
```

Comment:
@p3ck it will. As discussed in IRC, the test has a broken condition where the openstack credentials are not yet extracted if the beaker tests are run first. That needs to be resolved. I've added #620 to address the issue.